pkg/webui/console/views/gateway-add/connect.js

```diff
@@ -0,0 +1,53 @@
+// Copyright © 2022 The Things Network Foundation, The Things Industries B.V.
```

Comment:
Here you are creating a connected component. Isn't it better to use react hooks?

Reply:
Yes, I've refactored this to use hooks only.